api/backend.go

```diff
@@ -473,6 +473,18 @@ func (b *StatusBackend) NotifyUsers(message string, payload fcmlib.NotificationP
 	return err
 }
 
+// SendMessageToUsers sends a message to users, which can be used to display a push notification.
+func (b *StatusBackend) SendMessageToUsers(dataPayloadJSON string, tokens ...string) error {
```

Comment:
As we expose it on `StatusBackend` maybe we can call this method less general? `SendMessageToUsers` indicates sending messages not notifications.

```suggestion
func (b *StatusBackend) SendNotifications(dataPayloadJSON string, tokens ...string) error {
```

Reply:
Also what is this `dataPayloadJSON`. Don't we want to make it a struct here? If it does not matter what's insight (a bit strange), then let's at least comment what should be in this payload.

Reply:
> SendMessageToUsers indicates sending messages not notifications.

In a sense we're indeed no longer sending notifications to users (we changed from notifications API to messaging API), but rather a generic FCM message that is captured by the RN app and then used to create a local PN. That was the reason for the name. We can rename it to `SendDataNotification` which more closely reflects what it does.

Reply:
> Also what is this dataPayloadJSON. Don't we want to make it a struct here? If it does not matter what's insight (a bit strange), then let's at least comment what should be in this payload.

Yeah, this is up to status-react what they want to put in here, I added a comment giving a general overview of how it looks.

Reply:
In fact, the only reason this method doesn't live in status-react is that the FCM library that exists for Clojure doesn't support the methods we need (at some point we can probably just fork it and create an upstream PR).